Marlin/src/lcd/menu/menu_tramming.cpp

```diff
@@ -43,11 +43,9 @@ float z_measured[G35_PROBE_COUNT] = { 0 };
 static uint8_t tram_index = 0;
 
 bool probe_single_point() {
-  // In BLTOUCH HS mode, the probe travels in a deployed state.
-  // Users of Tramming Wizard might have a badly misaligned bed, so raise Z by the
-  // length of the deployed pin (BLTOUCH stroke < 7mm)
-  do_blocking_move_to_z((Z_CLEARANCE_BETWEEN_PROBES) + TERN0(BLTOUCH_HS_MODE, 7));
-  const float z_probed_height = probe.probe_at_point(screws_tilt_adjust_pos[tram_index], PROBE_PT_RAISE, 0, true);
+  do_blocking_move_to_z(Z_CLEARANCE_DEPLOY_PROBE);
```

Comment:
Why did this change to `Z_CLEARANCE_DEPLOY_PROBE` instead of `Z_CLEARANCE_BETWEEN_PROBES`?

Won't `probe_at_point` already move to deploy clearance if the probe needs to be deployed?

Reply:
When I left it as `Z_CLEARANCE_BETWEEN_PROBES`, after homing and after probing without `ASSISTED_TRAMMING_WAIT_POSITION`, the z axis moves down to `Z_CLEARANCE_BETWEEN_PROBES` then moves to the desired tramming point, and raises to `Z_CLEARANCE_DEPLOY_PROBE`.
With it changed to `Z_CLEARANCE_DEPLOY_PROBE`, it just moves to the desired tramming point without any extra z movement.

But do let me know if it is wrong.

Reply:
I am trying to think of what it would do with a FIX_MOUNT_PROBE that requires manual deployment high above the bed. Those would frequently have a deploy clearance of about 50mm.

Maybe just add another `TERN` here to treat BLTouch as a special case, since it stows between each point?

`do_blocking_move_to_z(TERN(BLTOUCH, Z_CLEARANCE_DEPLOY_PROBE, Z_CLEARANCE_BETWEEN_PROBES);`

Reply:
Ok, I can see that working. 👍